src/parsers/gas.js

```diff
@@ -119,19 +121,40 @@ export const defaultGasPriceFormat = (option, timeWait, value) => {
  * @param {Object} prices
  * @param {Number} gasLimit
  */
-export const parseTxFees = (gasPrices, priceUnit, gasLimit, nativeCurrency) => {
+export const parseTxFees = (
+  gasPrices,
+  priceUnit,
+  gasLimit,
+  nativeCurrency,
+  l1GasFeeOptimism
```

Comment:
🤮

Reply:
Do you have a suggestion?

Reply:
just complaining about how we're forced to do this change :)